tests/integration_tests/performance/test_net_tput.py

```diff
@@ -0,0 +1,290 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests the network throughput overhead added by Firecracker."""
+
+import json
+import time
+import statistics
+from subprocess import run
+import pytest
+
+import host_tools.network as net_tools
+
+RUNS = 1
+
+IPERF = 'iperf3'
+ITERATION_DURATION = 1
+WINDOW_SIZE = 10
```

Comment:
While a fixed window size gets more predictability you should also run tests without windows size. 
Please also put a comment or rename it to WINDOW_SIZE_MB that this is in MB, also on iperf duration what it means.

Reply:
Good catch, I will update the name and run TCP without window size too.